plash_cli/core.py

```diff
@@ -110,17 +110,20 @@ def login(
 def deploy(
     path:Path=Path('.'), # Path to project
     local:bool=False,    # Local dev
-    port:int=5002):      # Port for local dev
+    port:int=5002,       # Port for local dev
+    app_id:str=None):    # App ID that will be used as the subdomain in plash
+
     """🚀 Ship your app to production"""
```

Comment:
Nitpick, and not part of your changed code, but can we please make this a regular string not blockstring?